docs/FOR VALIDATORS/Join Mainnet.md

```diff
@@ -0,0 +1,133 @@
+# Join Mainnet
+
+## Mainnet
+You need to set the **genesis file** and **seeds**. If you need more information about past networks, check our [mainnet repo](https://github.com/imversed/mainnet). The table below gives an overview of all Mainnet Chain IDs. Note that, the displayed version might differ when an active Software Upgrade proposal exists on chain.
+
+| Chain ID | Description | Site | Version | Status |
+| --- | --- | --- | --- | --- |
+| `imversed_5555555-1` | imversed_5555555-1  | Imversed | v3.11 | `Live` |
+
+> **IMPORTANT:** If you join mainnet as a validator make sure you follow all the [security](https://docs.imversed.com/validators/security/security.html) recommendations!
+
+## Install `imversed`
+Follow the [installation](https://docs.imversed.com/validators/quickstart/installation.html) document to install the Imversed binary `imversed`.
+
+> Make sure you have the right version of `imversed` installed.
+
+### Save Chain ID
+We recommend saving the mainnet `chain-id` into your `imversed`'s `client.toml`. This will make it so you do not have to manually pass in the `chain-id` flag for every CLI command.
+
+> See the Official [Chain IDs](https://docs.imversed.com/users/technical_concepts/chain_id.html#official-chain-ids) for reference.
+
+```text
+imversed config chain-id imversed_9001-2
+```
+
+## Initialize Node
+We need to initialize the node to create all the necessary validator and node configuration files:
+
+```text
+imversed init <your_custom_moniker> --chain-id imversed_9001-2

```

Comment:
Change chain-id **imversed_9001-2** to **imversed_5555555-1**

Reply:
Updated...